apiserver/facades/client/application/application.go

```diff
@@ -386,6 +386,7 @@ func (api *APIBase) Deploy(args params.ApplicationsDeploy) (params.ErrorResults,
 	}
 
 	for i, arg := range args.Applications {
+		state.TranslateSpaceNameToID(api.backend, arg.EndpointBindings)
```

Comment:
Shouldn't we capture the return value and error here?

Reply:
turns out the data wasn't used, removed it and refactored TranslateSpaceNameToID for local state use.